### ISSUE TITLE
topo map source definitions are repeated

### ISSUE THREAD

**Issue by jariarkko** [jariarkko/psgeo#331]
Wondering if it would be helpful to have a separate definition of topo map sources, and then the individual websites could just declare if they want to use topo maps or not. Now there's some repetition in the defaults file.

Not urgent, ponder it for later.

**Comment by ralfstrandell** [comment]
Yes. Lots of repetition. It's quite annoying to scroll through all of it.
OTOH, this is not a problem for a third party managing but one site.
It is an issue for us, who have several sites in the same file.
Let's consider this for a june release.

**Comment by ralfstrandell** [comment]
we can move most of the stuff (not the show:true|false) from domain defaults into psgeosettings.js (cfg object).
--> next release
